remix/app/routes/_app.newsletter.update.$id.tsx

```diff
@@ -58,19 +78,31 @@ export default function Index() {
   }, [result]);
 
   return (
-    <div>
-      <Form method="put">
-        <div>
-          <label htmlFor="name">Name</label>
-          <input type="text" name="name" placeholder="name" />
-        </div>
-        <br />
-        <div>
-          <label htmlFor="content">Content</label>
-          <textarea name="content" placeholder="content" />
+    <section className="flex flex-col gap-4">
+      <header className="text-xl font-bold">
+        <h1>Update your Newsletter </h1>
+      </header>
+      <ValidatedForm
+        validator={validator}
+        defaultValues={newsletter}
+        method="put"
+      >
+        <div className="flex flex-col gap-6">
+          <FormInput
+            label={'Name'}
+            name={'name'}
+            placeholder={'name'}
```

Comment:
I think it would be a bit better if the `placeholder` can be capitalized like "Name" or "Please input the newsletter name here" :D

Reply:
Improved here: [b4eddfe](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/b4eddfee4c7915ecdf0fca12edc7b25d45db97d8)